iosHyperskillApp/iosHyperskillApp/Sources/Modules/StepQuiz/ViewData/StepQuizViewDataMapper.swift

```diff
@@ -35,10 +35,12 @@ final class StepQuizViewDataMapper {
             millisSinceLastCompleted: step.millisSinceLastCompleted
         )
 
-        let attemptLoadedState = state as? StepQuizFeatureStateAttemptLoaded
```

Comment:
It's simpler to try to unwrap state only ones as `attemptLoadedState` rather than separately in `quizName` and `feedbackHintText`:
```swift
let attemptLoadedState: StepQuizFeatureStepQuizStateAttemptLoaded? = {
    if case .attemptLoaded(let attemptLoadedState) = state {
        return attemptLoadedState
    }
    return nil
}()
```